lib/token_util.py

```diff
@@ -158,6 +161,22 @@ def sign_token(identity, issuer, kid, master_key, duration=None, scope=None):
         payload["exp"] = exp
     if scope:
         payload["scope"] = scope
+    # From https://pyjwt.readthedocs.io/en/stable/api.html
+    # jwt.encode(payload, key, algorithm="HS256", headers=None, json_encoder=None)
+    #   payload (dict), key (str), algorithm (str), headers (dict), json_encoder (json.JSONEncoder)
+    # From source code: https://github.com/jpadilla/pyjwt/blob/72ad55f6d7041ae698dc0790a690804118be50fc/jwt/api_jws.py
+    #     def encode(
+    #         self,
+    #         payload: bytes,
+    #         key: AllowedPrivateKeys | str | bytes,
+    #         algorithm: str | None = "HS256",
+    #         headers: dict[str, Any] | None = None,
+    #         json_encoder: type[json.JSONEncoder] | None = None,
+    #         is_payload_detached: bool = False,
+    #         sort_headers: bool = True,
+    #     ) -> str:
+    # From source code: https://github.com/jpadilla/pyjwt/blob/72ad55f6d7041ae698dc0790a690804118be50fc/jwt/utils.py
+    # If master_key is str, then it is encoded w/ utf-8:  value.encode("utf-8")
```

Comment:
I see that these lines are commented. Is there any specific reason why these lines are retained instead of completely removing them?

Reply:
To explain why bytes should be used or to use utf-8 if using str. Is it loo much, too long and confusing?

Reply:
> To explain why bytes should be used or to use utf-8 if using str. Is it loo much, too long and confusing?

Since I do not know the full technical context, it does seem a bit confusing because there are three URLs being referenced in the details regarding the `encode` method. Is it possible to describe the details concisely, given that the details seems rather important? Suggesting this because just looking at it the way it is does not make the detail quite obvious/evident.

Reply:
I changed it to a shorter explanation which can be important for future changes of the code
```
    # master_key should be `bytes`. `str` could cause value changes if was decoded not using utf-8.
    # The manual (https://pyjwt.readthedocs.io/en/stable/api.html) is incorrect to list `str` only.
    # The source code (https://github.com/jpadilla/pyjwt/blob/72ad55f6d7041ae698dc0790a690804118be50fc/jwt/api_jws.py)
    # shows `AllowedPrivateKeys | str | bytes` and if it is str, then it is encoded w/ utf-8:  value.encode("utf-8")
```